Makefile

```diff
@@ -315,6 +315,7 @@ SRC = \
 	src/Utilities/AssemblyHelper.cs \
 	src/Utilities/FileHelpers.cs \
 	src/Utilities/FNAInternalExtensions.cs \
+	src/Utilities/MarshalHelper.cs
```

Comment:
```suggestion
	src/Utilities/MarshalHelper.cs \
```

Reply:
Can you tell I'm not set up to run the makefile? I'm going to to try run this locally